### ISSUE TITLE
Outdated? 2020...

### ISSUE THREAD

**Issue by blausand** [masmu/pulseaudio-dlna#415]
Since **DLNA** got quite popular during the last two years i'd like to humbly ask if the repos can be updated easily to support current versions of `python`, `Debian`, `Ubuntu`, and `Linux Mint` by the current maintainers.
Maybe a short notice in the `readme.md` about whether pulseaudio-dlna is still the first choice to gain DLNA-playback on a system level might be helpful. _Thanks!_

**Comment by stuaxo** [comment]
@blausand I would consider making bugs in the upstream repos and linking back to this one.

**Comment by jejo86** [comment]
You can find [here](https://github.com/Cygn/pulseaudio-dlna/issues/21#issuecomment-849090064) the commands to build (the forked version) on **Linux Mint 20.1**.

Afterwards you would have to add the path of the built files to the path variable
```
echo "export PATH=\"$HOME/.local/bin:$PATH\"" >> ~/.bashrc
source ~/.bashrc
```

When this is done, you can start and use the project via the following commands
```
pulseaudio-dlna
pavucontrol 
```
I hope this helps.

**Comment by blausand** [comment]
Yeah, thanks. Probably i could do that. But people out there wouldn't follow as we all know.
In order to survive as a solution, this has to become a no-brainer IMHO.
Today, we see DLNA as the open standard threatened by Airplay and some other proprietary alternatives.
Let's vote for packaging `pulseaudio-dlna` with the vanilla installation to promote the open standard.
